packages/blade/src/components/Card/Card.tsx

```diff
@@ -16,6 +16,7 @@ import type { Elevation } from '~tokens/global';
 import type { SurfaceLevels } from '~tokens/theme/theme';
 import type { BoxProps } from '~components/Box';
 import { makeAccessible } from '~utils/makeAccessible';
+import { useVerifyAllowedChildren } from '~utils/useVerifyAllowedChildren/useVerifyAllowedChildren';
```

Comment:
export it from index?
```suggestion
import { useVerifyAllowedChildren } from '~utils/useVerifyAllowedChildren';
```

Reply:
If we export from index, it will be exposed to consumers.